core/trino-main/src/main/java/io/trino/security/AccessControl.java

```diff
@@ -366,6 +366,13 @@ default void checkCanSetViewAuthorization(SecurityContext context, QualifiedObje
      */
     void checkCanRenameMaterializedView(SecurityContext context, QualifiedObjectName viewName, QualifiedObjectName newViewName);
 
+    /**
+     * Check if identity is allowed to set the properties of the specified materialized view.
+     *
+     * @throws AccessDeniedException if not allowed
+     */
+    void checkCanSetMaterializedViewProperties(SecurityContext context, QualifiedObjectName materializedViewName, Map<String, Object> properties);
```

Comment:
Could you order commits (make `Support DEFAULT when setting mv properties` an earlier commit) in such a away that these new methods are not changed later within same PR?